griptape/drivers/structure_run/base_structure_run_driver.py

```diff
@@ -1,12 +1,15 @@
 from abc import ABC, abstractmethod
 
+from attr import field
 from attrs import define
 
 from griptape.artifacts import BaseArtifact
 
 
 @define
 class BaseStructureRunDriver(ABC):
```

Comment:
Can you add a docstring to the class with a brief description of the `env` field?